10.9-libcxx/stable/main/finkinfo/devel/meson-py.info

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; tab-width: 4 -*-
+Info4: <<
+Package: meson-py%type_pkg[python]
+Version: 0.53.2
+Revision: 1
+Maintainer: Leigh Smith <leigh@leighsmith.com>
+Type: python (3.6 3.7 3.8)
+# setuptools needed because meson uses pkg_resources package.
+Depends: <<
+	python%type_pkg[python],
+	setuptools-tng-py%type_pkg[python]
+<<
+Source: https://files.pythonhosted.org/packages/49/d7/38a71de6adaa7a25a72587e69429ce78e137f4d545c7516341aa081f43e7/meson-%v.tar.gz
+Source-Checksum: SHA256(3e8f830f33184397c2eb0b651ec502adb63decb28978bdc84b3558d71284c21f)
```

Comment:
```suggestion
	ninja (>= 1.7-1)
<<
Source: https://files.pythonhosted.org/packages/source/m/meson/meson-%v.tar.gz
Source-Checksum: SHA256(0a1ae2bfe2ae14ac47593537f93290fb79e9b775c55b4c53c282bc3ca3745b35
```
We are generally using the human-readable PyPI links now (SHA256 for 0.55.0).
My impression was that `ninja` is needed for any actual builds with `meson` (the Homebrew package also has it as a dependency), but I may stand corrected and perhaps it's just a `Recommends`.